src/bug-filing/common/issue-filing-url-string-utils.ts

```diff
@@ -1,60 +1,50 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 import { endsWith } from 'lodash';
-
-import { EnvironmentInfo } from '../../common/environment-info-provider';
-import { title } from '../../content/strings/application';
 import { CreateIssueDetailsTextData } from './../../common/types/create-issue-details-text-data';
 
 export type IssueUrlCreationUtils = {
     getTitle: (data: CreateIssueDetailsTextData) => string;
-    getFooter: (info: EnvironmentInfo) => string;
-    collapseConsecutiveSpaces: (input: string) => string;
-    formatAsMarkdownCodeBlock: (input: string) => string;
     getSelectorLastPart: (selector: string) => string;
     standardizeTags: (data: CreateIssueDetailsTextData) => string[];
     appendSuffixToUrl: (url: string, suffix: string) => string;
 };
 
-export const IssueFilingUrlStringUtils: IssueUrlCreationUtils = {
-    getTitle: (data: CreateIssueDetailsTextData): string => {
-        const standardTags = IssueFilingUrlStringUtils.standardizeTags(data);
+export const createUtils = (): IssueUrlCreationUtils => {
```

Comment:
>export [](start = 0, length = 6)

is exporting this necessary?

Reply:
Same as the comment for the indirection. It makes it easier to reuse functions inside the object being created. Look at `getTitle` for example, it use `standardizeTags`. For me it reads better like this instead of `IssueFilingUrlStringUtils.standardizeTag`

Reply:
That answers why you'd use a separate function for it, but not why the function itself has to be exported (as opposed to it being a non-exported function and only the result of calling it is exported).

Reply:
No misunderstanding on your side. You're totally right, we don't need to export the `createUtils` function, which means you're probably right about not actually need the function in the first place. That just slip my mind. I wrote it like that because of other files having the same coding style where I need to expose the create function in order to pass mocked dependencies; that's clearly not this case so I'll update it following your comments.

Reply:
Updated. Does this work better?